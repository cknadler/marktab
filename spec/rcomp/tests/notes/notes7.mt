# note transition
1:1 / 1:4
