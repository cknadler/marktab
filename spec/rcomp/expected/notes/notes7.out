e|-1-/-4-|
B|-------|
G|-------|
D|-------|
A|-------|
E|-------|
